qutip/solver/heom/bofin_solvers.py

```diff
@@ -894,7 +905,7 @@ def _grad_next_fermionic(self, he_n, k):
             )
         return op
 
-    def _rhs(self):
+    def _HEOMRHS(self):
```

Comment:
```suggestion
    def _calculate_rhs_ados(self):
        """Calculate the ADOs contribution to the RHS."""
```
The original naming here was not that clear.